src/crypto.test.ts

```diff
@@ -20,6 +53,23 @@ describe("deriveChildSeed", () => {
   });
 });
 
+describe("encodeBigint", () => {
+  it("should correctly encode bigints", () => {
+    const maxint = "18446744073709551615";
```

Comment:
We define `maxint` in another place (or two?) as well. Let's move it to a global const.